docs/development.md

```diff
@@ -38,7 +38,7 @@ go run . serve
 
 Install prerequisites:
 
-- [CMake](https://cmake.org/download/)
+- [CMake](https://cmake.org/download/) -- Not used on Windows ARM 
```

Comment:
This is a separate section so maybe it'll be more clear if the heading is `Windows (x86)`

Reply:
Good point - I need to spell out that you need the gcc wrapper for clang on windows arm to be able to build cgo.  That got dropped from the guide in the shuffle.